lib/pong_game_web/channels/game.ex

```diff
@@ -0,0 +1,17 @@
+defmodule PongGameWeb.Game do
+  use GenServer
+
+  def start_link() do
+    GenServer.start_link(__MODULE__, nil, name: {:global, :default_game})
+  end
+
+  def init(_) do
+    {:ok, "hello world"}
```

Comment:
🐘 

If you want to make the state look like something normal rather than "hello world", you could replace it with a map

It wanted to improve this so it's a better example of showing how the different parts fit together, you could:
- replace the initial state with something like `%{count: 0}`
- increment the count every time `handle_call(:test, ...)` is called
- maybe rename `:test` to something else, as the word test appears a lot in the code (to make it easier to rename later)
- make the `handle_call` return the entire state and send that to the client. I think this is what the game will have to do in the future